libretroshare/src/services/p3wire.cc

```diff
@@ -31,6 +31,23 @@
 RsWire *rsWire = NULL;
 
 
+uint32_t RsWirePulse::ImageCount()
+{
+	if (!mImage4.empty()) {
```

Comment:
The logic seems risky here. One can craft a msg with mImage3=nullptr and mImage4 not empty. But still the number of images will not be 4.

Reply:
made this more robust